packages/tinygo/build.sh

```diff
@@ -55,8 +57,8 @@ termux_pkg_auto_update() {
 	local uptime_s="${uptime_now//.*}"
 	local uptime_h_limit=1
 	local uptime_s_limit=$((uptime_h_limit*60*60))
-	[[ -z "${uptime_s}" ]] && e=1
-	[[ "${uptime_s}" == 0 ]] && e=1
+	[[ -z "${uptime_s}" ]] && [[ "$(uname -o)" != "Android" ]] && e=1
```

Comment:
I don't think this will work. `termux_pkg_auto_update` will be executed on the GitHub Action Runner, whose system is Ubuntu (GNU/Linux).

Reply:
Ahhh... I misunderstand it...